client/components/card/README.md

```diff
@@ -32,7 +32,8 @@ Name | Type | Default | Description
 --- | --- | --- | ---
 `className` | `string` | null | Adds CSS classes.
 `href` | `string` | null | URL of the card. Adds a right chevron icon.
-`tagName` | `string` | null | Allows you to control the tag name of the card wrapper (only if `href` is not specified).
+`fakeLink` | `bool` | false | Used for cards without a `href` that need to display as a link. The card will render as an accessible button with the same visual appearance as a link (including the right chevron icon).
```

Comment:
🤔 this does seem like the simplest way to get this behavior. What if we called it `clickable` rather than `fakeLink`? To me that seems more clear; then `href` is more like a shortcut to adding `clickable` and a URL. It might be worth noting in the docs that if you specify `clickable` you should also have `href` or `onClick` set?

Reply:
Not sure about `clickable` because there can be clickable cards (e.g. regular button cards) that don't want this extra styling.

How about something like `displaysAsLink` instead, but also require the user to specify the tag?  So for example, `<Card tagName="button" displaysAsLink onClick=...` would make a button that looks like a link.

> It might be worth noting in the docs that if you specify clickable you should also have href or onClick set?

Agreed, since it's primarily for buttons, I'll definitely mention the `onClick`.